### PR TITLE
Add CSH Plug Panel

### PR DESCRIPTION
**Pull request by liam-middlebrook**
If you think about it, this is the ideal place for this panel. It's right under the most popular options that people want 99% of the time they visit members. But still above the fold so it will get the exposure it deserves.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,6 +64,18 @@ <h3 class="panel-title">Popular</h3>
...
                 </div>
+                <div class="panel panel-primary">
+                    <div class="panel-heading">
+                        <h3 class="panel-title"><a href="https://plug.csh.rit.edu">Advertisements by CSH: Plug</a></h3>
+                    </div>
+                    <div class="row">
+                        <div class="center">
+                           <a href="https://plug.csh.rit.edu"><img src="https://plug.csh.rit.edu/data"></a>
+                        </div>
+                        <!-- col-md-2 col-sm-4 col-xs-6 center -->
```
Comment (stevenmirabito) [inline index.html]: Remove this comment as the tag it refers to doesn't exist

File: index.html [hunk 2]
```diff
@@ -64,6 +64,18 @@ <h3 class="panel-title">Popular</h3>
                     </div>
                     <!-- row -->
                 </div>
+                <div class="panel panel-primary">
+                    <div class="panel-heading">
+                        <h3 class="panel-title"><a href="https://plug.csh.rit.edu">Advertisements by CSH: Plug</a></h3>
+                    </div>
+                    <div class="row">
```
Comment (stevenmirabito) [inline index.html]: You probably want a `panel-body` here instead of a `row`.

File: index.html [hunk 3]
```diff
@@ -64,6 +64,18 @@ <h3 class="panel-title">Popular</h3>
...
+                        <div class="center">
+                           <a href="https://plug.csh.rit.edu"><img src="https://plug.csh.rit.edu/data"></a>
+                        </div>
+                        <!-- col-md-2 col-sm-4 col-xs-6 center -->
+                    </div>
+                    <!-- row -->
```
Comment (stevenmirabito) [inline index.html]: Update this comment if you change the class as suggested above.

**Comment by liam-middlebrook** [comment]
Updated with requested changes.

**Pull request merged**